controllers/articles-controllers.js

```diff
@@ -1,4 +1,12 @@
-const { fetchArticleById } = require("../models/articles.models");
+const { fetchArticles, fetchArticleById } = require("../models/articles.models");
+
+exports.getArticles = (req, res, next) => {
+  fetchArticles()
+    .then(({ rows }) => {
+      res.status(200).send({ rows });
```

Comment:
lets not send back articles on a key of rows lets send articles back on a key of articles 🙂